src/Storage/Helpers/InstanceHelper.cs

```diff
@@ -106,22 +106,23 @@ public static string GetSBLStatusForCurrentTask(Instance instance)
                 {
                     return "Archived";
                 }
-                else if (instance.Process.CurrentTask.AltinnTaskType.Equals("confirmation", StringComparison.OrdinalIgnoreCase))
+                else if (instance.Process?.CurrentTask?.AltinnTaskType != null)
                 {
-                    return "Confirmation";
-                }
-                else if (instance.Process.CurrentTask.AltinnTaskType.Equals("feedback", StringComparison.OrdinalIgnoreCase))
-                {
-                    return "Feedback";
-                }
-                else if (instance.Process.CurrentTask.AltinnTaskType.Equals("signing", StringComparison.OrdinalIgnoreCase))
-                {
-                    return "Signing";
-                }
-                else
-                {
-                    return "FormFilling";
+                    if (instance.Process.CurrentTask.AltinnTaskType.Equals("confirmation", StringComparison.OrdinalIgnoreCase))
```

Comment:
could add a local variable string TaskType = instance.Process.CurrentTask.AltinnTaskType; 
to make reading the following if checks a bit easier.

Reply:
Done